NuGet/Realm.Database/Realm.Database.nuspec

```diff
@@ -17,8 +17,18 @@
     <tags>Realm database db persistence sqlite</tags>
     <dependencies>
       <dependency id="Fody" version="1.29.4"/>
+      <dependency id="Realm.DataBinding" version="1.0.0"/>
       <dependency id="DotNetCross.Memory.Unsafe" version="0.2.2"/>
```

Comment:
Let's leave it for later as this one's already huge and would prefer not to add more points where things might break :)